inst/rstudio/r.snippets

```diff
@@ -0,0 +1,70 @@
+snippet tm.mod.new
```

Comment:
snippets work in RSP1.2, but not in RStudio 1.3.776 on Mac

Reply:
RStudio 1.3.776 is a preview version

Reply:
We should be fine with our version, will check that one later.